### PR TITLE
Added user verification

### PR DESCRIPTION
**Pull request by ryanlioy**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ var cookieParser = require('cookie-parser');
 var logger = require('morgan');
 var mongoose = require('mongoose');
 var fs = require('fs');
+var user = require("./models/user");
```
Comment (votrungquan1999) [inline app.js]: why do you need to import user here?

File: routes/users.js [hunk 2]
```diff
@@ -1,9 +1,56 @@
 var express = require('express');
 var router = express.Router();
+var mongoUser = require('../models/user').MongoModel;
+var joiUser = require('../models/user').JoiModel;
+
+/*
+Function for determining if a username is already
+in the database
+*/
+function isUniqueUser(uname){
```
Comment (votrungquan1999) [inline routes/users.js]: you don't need this function any more right?

**Review by votrungquan1999 — COMMENTED** [review]
@Ovil101 can you check the comment i made?

File: app.js (hunk 1 above)
Comment (ryanlioy) [reply app.js]: The would be from what we were trying to do earlier and forgot to remove.

File: routes/users.js (hunk 2 above)
Comment (ryanlioy) [reply routes/users.js]: Correct, forgot to remove it before making the pull request.

**Pull request merged**